includes/admin/views/omise-page-settings.php

```diff
@@ -136,7 +136,9 @@
 
 		<h3><?php _e( 'Payment Methods', 'omise' ); ?></h3>
 		<?php if ($settings['account_country']) : ?>
-			<p><?php _e( 'The table below is a list of available payment methods that you can enable in your WooCommerce store.', 'omise' ); ?></p>
+			<!--  account_country exists when API key is valid -->
+			<?php  $available_payment_methods = Omise_Capabilities::retrieve()->get_available_payment_methods();  ?>
```

Comment:
Are we sure this won't throw error when keys are not present? Just being extra cautious after the incident.

Reply:
yeah i think is no longer valid check like now if key is eg. expired is should got any error on this one 🤔 
any suggestions 🙏

Reply:
You can put this line into the `render()` method of `Omise_Page_Settings`. If the `Omise_Capabilities::retrieve()` gives `null`, you assign `null` or `[]` whichever should be the datatype of `$available_payment_methods` and pass it to the view.

Reply:
Here update follow suggestion 🙏  -> https://github.com/omise/omise-woocommerce/pull/290/commits/d4b2ecf85a696924b084332e61dd81293cc642da
Thanks you very much 🙇